tests/benchmark/runtime/client.go

```diff
@@ -0,0 +1,70 @@
+package runtime
+
+import (
+	"context"
+
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/client"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/crypto/signature"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/types"
+)
+
+const (
+	methodAccoutsMint      = "benchmarks.accounts.Mint"
+	methodAccountsTransfer = "benchmarks.accounts.Transfer"
+)
+
+type V1 interface {
+	AccountsMint(ctx context.Context, signer signature.Signer, nonce uint64, mint *AccountsMint) error
+
+	AccountsTransfer(ctx context.Context, signer signature.Signer, nonce uint64, transfer *AccountsTransfer) error
+}
+
+type v1 struct {
+	rc client.RuntimeClient
+}
+
+// Implements V1.
+func (a *v1) AccountsMint(ctx context.Context, signer signature.Signer, nonce uint64, mint *AccountsMint) error {
+	info, err := a.rc.GetInfo(ctx)
+	if err != nil {
+		return err
+	}
+
+	tx := types.NewTransaction(nil, methodAccoutsMint, mint)
+	tx.AppendAuthSignature(signer.Public(), nonce)
+	stx := tx.PrepareForSigning()
+	if err = stx.AppendSign(info.ChainContext, signer); err != nil {
+		return err
+	}
+
+	if _, err = a.rc.SubmitTx(ctx, stx.UnverifiedTransaction()); err != nil {
```

Comment:
Can you change these to use the transaction builder (just to be consistent with other modules)?

Reply:
changed